sdk/src/main/java/co/omise/android/api/EndPoint.kt

```diff
@@ -61,6 +59,9 @@ abstract class Endpoint {
     }
 
     companion object {
+        /**
+         * Class containing all the information "VAULT" endpoint.
```

Comment:
information (for the?) "VAULT" endpoint?

Reply:
Correct 👍